lvk/vulkan/VulkanClasses.cpp

```diff
@@ -5524,7 +5447,7 @@ void lvk::VulkanContext::generateMipmap(TextureHandle handle) const {
 
   LVK_ASSERT(tex->vkImageLayout_ != VK_IMAGE_LAYOUT_UNDEFINED);
   const lvk::VulkanImmediateCommands::CommandBufferWrapper& wrapper = immediate_->acquire();
-  tex->generateMipmap(wrapper.cmdBuf_);
+  tex->generateMipmap(wrapper.cmdBuf_, debugUtilsAvailable_);
```

Comment:
No need to add an extra param, just check if actual debug utils functions are `nullptr`.